source/frontend/src/components/Table/DateCell.tsx

```diff
@@ -1,7 +1,8 @@
 import { CellProps } from 'react-table';
-import { prettyFormatDate, prettyFormatDateTime } from 'utils';
+import { prettyFormatDate, prettyFormatDateTime, stringToFragment } from 'utils';
 
-export const DateCell = ({ cell: { value } }: CellProps<Date, string>) => prettyFormatDate(value);
+export const DateCell = ({ cell: { value } }: CellProps<any, string | Date | undefined>) =>
+  stringToFragment(prettyFormatDate(value));
 
-export const DateTimeCell = ({ cell: { value } }: CellProps<Date, string>) =>
-  prettyFormatDateTime(value);
+export const DateTimeCell = ({ cell: { value } }: CellProps<any, string | Date | undefined>) =>
+  stringToFragment(prettyFormatDateTime(value));
```

Comment:
Upgrading typescript exposed a series of typing errors. This one is a common-one. All table cell renderers should return a valid JSX element. For some reason old typescript was happy with just strings, new typescript complains. So I created a helper method to return a React fragment wrapping the string. Basically `<>YOUR STRING HERE</>`